src/main/java/com/example/onlinebookstore/dto/category/UpdateCategoryRequestDto.java

```diff
@@ -0,0 +1,7 @@
+package com.example.onlinebookstore.dto.category;
+
+public record UpdateCategoryRequestDto(
+        String name,
+        String description
```

Comment:
use the same validators that you use for creation. Should you validate the size?